dragonfly/room2d.py

```diff
@@ -769,7 +769,10 @@ def to_honeybee(self, multiplier=1, tolerance=0.01):
                 floor_to_ceiling_height at which adjacent Faces will be split.
                 This is also used in the generation of Windows. Default: 0.01,
                 suitable for objects in meters.
+            add_plenums: Boolean to auto-generate floor and ceiling plenums for the Room
+                (Default: False).
```

Comment:
Missing docstring for plenum height.

Reply:
I'll remove the plenum_height argument and set it from parent's floor to floor.